packages/backend/src/app.ts

```diff
@@ -265,6 +266,7 @@ export interface AppServices {
   paymentMethodHandlerService: Promise<PaymentMethodHandlerService>
   ilpPaymentService: Promise<IlpPaymentService>
   localPaymentService: Promise<LocalPaymentService>
+  authServiceClient: AuthServiceClient
```

Comment:
```suggestion
  authServiceClient: Promise<AuthServiceClient>
```

Reply:
Technically I dont think I need this? Although I realize everything else is a promise but its driven by the factory function we pass into the singleton. I have:

```ts
  container.singleton('authServiceClient', () => {
    return new AuthServiceClient(config.authServiceApiUrl)
  })
```

I didn't make that `async` since it didn't need to be, hence the `AuthServiceClient` and not a promise fort he type here. I do see that we return a promise elsewhere where it doesnt seem like we need to. Recent in memory cache for example.

```ts
  container.singleton('tenantCache', async () => {
    return createInMemoryDataStore(config.localCacheDuration)
  })
```

So either I'm missing something or we've just done it this way because the other ones looked like this?